### PR TITLE
Adding duck label, per sources spec

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

Fixes #

## Proposed Changes

- adding `duck.knative.dev/source` from the which was missing here 
-
-

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

File: config/201-clusterrole.yaml [hunk 1]
```diff
@@ -17,6 +17,7 @@ kind: ClusterRole
 metadata:
   name: ceph-controller
   labels:
+    duck.knative.dev/source: "true"
```
Comment (lberk) [inline config/201-clusterrole.yaml:20]: Do we need this in the clusterrole? I was under the impression we just needed it in the source yaml

File: config/300-cephsource.yaml [hunk 2]
```diff
@@ -18,6 +18,7 @@ metadata:
   labels:
     ceph.eventing.knative.dev/release: devel
     eventing.knative.dev/source: "true"
+    duck.knative.dev/source: "true"
```
Comment (n3wscott) [inline config/300-cephsource.yaml:21]: this is good.

File: config/201-clusterrole.yaml (hunk 1 above)
Comment (n3wscott) [reply config/201-clusterrole.yaml:20]: This is not needed. We already have the source observer agg rule here: https://github.com/knative-sandbox/eventing-ceph/blob/main/config/201-clusterrole.yaml#L101

This change results in the source ducktype taking the RBAC rights of the controller, which we do not want.

/hold

File: config/300-cephsource.yaml (hunk 2 above)
Comment (n3wscott) [reply config/300-cephsource.yaml:21]: https://github.com/knative-sandbox/discovery/blob/main/config/knative/sources.duck.knative.dev.yaml#L23

**Comment by matzew** [comment]
/unhold
addressed comment, thanks @n3wscott

**Review by n3wscott — APPROVED** [review]
/lgtm

**Pull request merged**